docs/layout-props.md

```diff
@@ -301,6 +301,18 @@ See https://developer.mozilla.org/en-US/docs/Web/CSS/bottom for more details of
 
 ---
 
+### `columnGap`
+
+`columnGap` property sets the size of the gap (gutter) between an element's columns. It works similar to `column-gap` in CSS, but in React Native you must use points. Percentage unit is not supported.
+
+See https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap for more details.
```

Comment:
```suggestion
`columnGap` works like `column-gap` in CSS. Only pixel units are supported in React Native. See https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap for more details.
```

Reply:
> pixels != points

oh, I thought points were RN's way of saying device-independent pixels. Is that not the case? e.g. https://reactnative.dev/docs/layout-props#width

Reply:
okay, just realized you mean `points` = pt unit. i.e. 1pt = 1/72nd of 1in 😅. yeah, indeed that's not the case. But not sure we can call it pixel units as its value doesn't end with `px` suffix. Although it behaves like CSS pixel though 😅 (device independent). Maybe I am overthinking. I'll just push the commit. We can discuss it somewhere else.